README.rst

```diff
@@ -65,10 +72,30 @@ ordering of admin items. More information is available in the `django-admin-tool
 docs <http://django-admin-tools.readthedocs.io/en/latest/dashboard.html#>`_.
 
 
+For frontend development, you can activate a webpack dev server with hot reload
+using::
+
+
+    npm start
```

Comment:
We could maybe write a shell script, but imo not worth it. My intent with the different commands was to give folks who don't need live dev for the frontend an ability to build the styles once and then move on to doing python development, so that you don't need two servers running.

If you've just pulled the repo, none of these commands are required because `npm install` also calls `npm run build:qa` - you just need to call them if you made changes. Is that what you mean by required?

Reply:
Ah, that was the piece I was missing! I think you should clarify that in the directions, then. Is that what you meant by "frontend development" ?